PROJECT.md

```diff
@@ -2,34 +2,34 @@
 
 Each project repository **requires**, at a minimum:
 
-1. Readme -- `README.md`
-1. License -- `LICENSE`
-1. Contributing Guidelines -- `.github/CONTRIBUTING.md`
-1. Issue Template -- `.github/ISSUE_TEMPLATE.md`
+1. Readme — `README.md`
+1. License — `LICENSE`
+1. Contributing Guidelines — `.github/CONTRIBUTING.md`
+1. Issue Template — `.github/ISSUE_TEMPLATE.md`
 
-## Readme
+## Readme File
 
-Each **readme** follows a [`README.md` template](./templates/README.md)
+Each [README](https://en.wikipedia.org/wiki/README) file follows a [`README.md` template](./templates/README.md).
 
 ## License
 
-[GPLv3](http://gplv3.fsf.org/)--or
+[GPLv3](http://gplv3.fsf.org/)—or
 [AGPLv3](http://www.gnu.org/licenses/agpl-3.0.html) ([_why Affero?_
-](http://www.gnu.org/licenses/why-affero-gpl.html))--is our _preferred_
+](http://www.gnu.org/licenses/why-affero-gpl.html))—is our _preferred_
 **license** for code, and [Creative Commons Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)](https://creativecommons.org/licenses/by-sa/4.0/)
 for creative works and documentation.
 
 We also use a [License and Copyright block](#license--copyright-readme-block),
-added to our READMEs, add a [LICENSE](/LICENSE) to the repo, and occasionally
+added to our READMEs.  We also add a [LICENSE](/LICENSE) file to the repo, and occasionally
 include a [Code Header License block](#code-header-license-block).
 
-### License & Copyright README block
+### License & Copyright block for README files
 
 ```markdown
 ## License & Copyright
 
-Copyright (C) <year> Data Together
-This program is free software: you can redistribute it and/or modify it under
+Copyright (C) <year> Data Together.
```

Comment:
Hey @mhucka -- this is the part that I think we shouldn't change, this is verbatim how FSF reccomends acknowledging GPL:

>  <one line to give the program's name and a brief idea of what it does.>
>   Copyright (C) <year>  <name of author>
>
>    This program is free software: you can redistribute it and/or modify
>    it under the terms of the GNU General Public License as published by
>    the Free Software Foundation, either version 3 of the License, or
>    (at your option) any later version.

https://www.gnu.org/licenses/gpl.html